apps/meteor/ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.ts

```diff
@@ -43,7 +43,7 @@ export class VisitorInactivityMonitor {
 			name: this._name,
 			schedule: (parser) => parser.cron(everyMinute),
 			job: () => {
-				Promise.await(this.handleAbandonedRooms());
+				return this.handleAbandonedRooms();
```

Comment:
have you checked if `SyncedCron` supports `Promises`? I mean, it tracks how much time a cronjob has run, so it kinda waits for the function to execute.. 

what if we let this `Promise.await` until we take a look at the `SyncedCron` package?

Reply:
you are right I thought this one was from agenda, the meteor package doenst handle async functions